stories/features/welcome/page/screens/searchBox.tsx

```diff
@@ -6,7 +6,8 @@
 import * as React from 'react'
 
 // Feature-specific components
-import { Content, Title, Paragraph } from '../../../../../src/features/welcome/'
+import { Content, Title, Paragraph, SelectGrid } from '../../../../../src/features/welcome/'
+import { SelectBox } from '../../../../../src/features/shields'
```

Comment:
Tentatively using selectBox from shields - will finalize component structure after migration of features/stories to `brave-core`